xbmc/windowing/gbm/GBMUtils.cpp

```diff
@@ -103,3 +105,79 @@ void CGBMUtils::ReleaseBuffer()
   m_bo = m_next_bo;
   m_next_bo = nullptr;
 }
+
+// --- CGBMBufferObject ----------------------------------------------------
```

Comment:
How about a new source file so this doesn't become a dumping ground for GBM classes?